<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>posadovciForm</class>
 <widget class="QWidget" name="posadovciForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Введення працівників</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tke/tke_rc">
    <normaloff>:/pictures/pictures/cube.png</normaloff>:/pictures/pictures/cube.png</iconset>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableWidget" name="tableWidget_posadovci">
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string>id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Посада</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Прізвище</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ім'я</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>По батькові</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Підпис в док.</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="savePbtn">
         <property name="text">
          <string>Зберегти</string>
         </property>
         <property name="icon">
          <iconset resource="tke/tke_rc">
           <normaloff>:/pictures/pictures/floppy.png</normaloff>:/pictures/pictures/floppy.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>12</width>
           <height>12</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelPbtn">
         <property name="text">
          <string>Закрити</string>
         </property>
         <property name="icon">
          <iconset resource="tke/tke_rc">
           <normaloff>:/pictures/pictures/button_cancel.png</normaloff>:/pictures/pictures/button_cancel.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>12</width>
           <height>12</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="tke/tke_rc"/>
 </resources>
 <connections/>
</ui>
